scipy/stats/tests/test_multicomp.py

```diff
@@ -0,0 +1,133 @@
+import copy
+
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+
+from scipy import stats
+from scipy.stats._multicomp import pvalue_dunnett, DunnettResult
+
+
+class TestDunnett:
+
+    @pytest.mark.parametrize(
+        'rho, n_groups, df, statistic, pvalue, alternative',
+        [
+            # From Dunnett1995
+            # Tables 1a and 1b pages 1117-1118
+            (0.5, 1, 10, 1.81, 0.05, "one-sided"),  # different than two-sided
```

Comment:
`alternatives` is documented as accepting `{'two-sided', 'less', 'greater'}`, not `'one-sided'`.